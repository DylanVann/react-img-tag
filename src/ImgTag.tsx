import React, { SFC } from 'react'
import { imgTagString, ImgTagProps } from './imgTagString'

export const ImgTag: SFC<ImgTagProps> = props => (
    <span
        dangerouslySetInnerHTML={{
            __html: imgTagString(props),
        }}
    />
)
