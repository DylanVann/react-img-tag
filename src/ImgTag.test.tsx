import React from 'react'
import { ImgTag } from './ImgTag'
import renderer from 'react-test-renderer'

it('renders', () => {
    const tree = renderer.create(<ImgTag />).toJSON()
    expect(tree).toMatchSnapshot()
})

it('renders with src', () => {
    const tree = renderer.create(<ImgTag src="test.png" />).toJSON()
    expect(tree).toMatchSnapshot()
})
