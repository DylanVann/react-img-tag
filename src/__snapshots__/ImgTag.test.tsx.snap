// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<span
  dangerouslySetInnerHTML={
    Object {
      "__html": "<img src=\\"\\" alt=\\"\\" />",
    }
  }
/>
`;

exports[`renders with src 1`] = `
<span
  dangerouslySetInnerHTML={
    Object {
      "__html": "<img src=\\"test.png\\" alt=\\"\\" />",
    }
  }
/>
`;
